index.bs

```diff
@@ -7773,7 +7773,8 @@ partial dictionary MLOpSupportLimits {
 </div>
 
 ### transpose ### {#api-mlgraphbuilder-transpose}
-Permute the dimensions of the input tensor according to the *permutation* argument.
+Permute the dimensions of the input tensor.
```

Comment:
Should we keep mentioning `permutation` by using {{MLTransposeOptions/permutation}}?

Reply:
SGTM. I think I just removed it as obvious/redundant, but added back in. 8b065cb331c7b00fa1c65bc84e4540db8c59d279